mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -283,6 +283,39 @@
     <!-- Text for the button to not request notification permission on the device and dismiss the dialog -->
     <string name="onboarding_home_enable_notifications_negative_button">Not now</string>
 
+    <!-- Juno first user onboarding flow experiment -->
+    <!-- Title for set firefox as default browser screen.
+        The first parameter is the name of the app defined in app_name (for example: Fenix) -->
+    <string name="juno_onboarding_default_browser_title" tools:ignore="UnusedResources">Make %s your go-to browser</string>
+    <!-- Description for set firefox as default browser screen.
+        The first parameter is the string with key "juno_onboarding_default_browser_description_link_text", that says "privacy notice" in en-* -->
+    <string name="juno_onboarding_default_browser_description" tools:ignore="UnusedResources">Firefox puts people over profits and defends your privacy by blocking cross-site trackers.\n\nLearn more in our %s.</string>
```

Comment:
You need to fix the string in the fenix repository (no need to use a new ID in this case, since the strings are on hold). This repository only reads strings from there.

Reply:
Thanks! created a [PR](https://github.com/mozilla-mobile/firefox-android/pull/1397) for this, will update here as that's merged.